packages/scripts/src/commands/app-data/convert/processors/flowParser/parsers/default.parser.spec.ts

```diff
@@ -1,3 +1,135 @@
-describe("default Parser", () => {
-  it("TODO - add tests", () => expect(true).toEqual(true));
+import { FlowTypes } from "data-models";
+import { DefaultParser } from "./default.parser";
+
+const testInputSources = {
+  data_list: { test_data_list: [{ id: 1 }, { id: 2 }, { id: 3 }] },
+};
+
+describe("Default Parser", () => {
+  it("Cleans field values - removes whitespace within empty strings", () => {
+    const parser = new DefaultParser({ processedFlowHashmap: testInputSources } as any);
```

Comment:
Rather than declaring this for each spec test, you can declare it once at the start of the `specDefinitions` in the `describe` function. See line 47 of `packages/scripts/src/commands/app-data/convert/processors/flowParser/parsers/data_list.parser.spec.ts` for an example. In your case you want the parser to be instantiated as a `new DefaultParser()`, but you can use the function `getTestFlowParserProcessor`, which is imported into `data_list.parser.spec.ts`, to get an entity of the right type to pass to `new DefaultParser()`. Then you can get rid of `testInputSources`, which I don't think is serving any purpose. See if you can figure out how to declare `parser: DefaultParser` at the top of the spec tests so that it doesn't need to be declared within each one, something like this:

```ts
describe("Default Parser", () => {
  // Instantiate parser here

  it("does something", () => {
    const output = parser.run({

   ... 

```